server/routes/incidentio/tasks.py

```
@@ -0,0 +1,411 @@
+                            (
+                                user_id, org_id, incident_id, "incidentio", alert_db_id,
+                                fields["incident_name"], service, normalized_severity,
+                                "primary", 1.0, json.dumps(alert_metadata),
+                            ),
+                        )
+                        cursor.execute(
+                            "UPDATE incidents SET affected_services = ARRAY[%s] WHERE id = %s",
+                            (service, incident_id),
+                        )
+                        conn.commit()
+                    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)